package.json

```diff
@@ -11,13 +11,6 @@
   "main": "dist/index.js",
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
-  "exports": {
```

Comment:
Did you mean to delete all this?

Reply:
Hm... I did not. I only meant to remove `src/` from the exports. I'll fix this and the `test:playwright` scripts I apparently added...